synapse/http/server.py

```diff
@@ -629,7 +629,7 @@ def _encode_json_bytes(json_object: Any) -> Iterator[bytes]:
 
 
 def respond_with_json(
-    request: Request,
+    request: SynapseRequest,
```

Comment:
This doesn't look like it needs a `SynapseRequest`? I'm guessing that's coming in a different PR?

Reply:
Ah, yes it is.